app/views/articles/_search_results.html.erb

```diff
@@ -22,7 +22,7 @@
         <%= render 'search_header' %>
         <%= render 'research_starters' %>
         <%= render_document_index @response.documents %>
-        <%= render 'alternate_catalog' if show_alternate_catalog? %>
+        <%= render 'alternate_catalog', close: true if show_alternate_catalog? %>
```

Comment:
This might be a future problem and not applicable to this PR, but I think this is a class called alert-dismissible in bootstrap5 and we are going to get rid of the close button in bootstrap 5 so I am wondering if we should rename close to dismissible?